<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<title>Unexpected I/O log</title>
<meta http-equiv="cache-control" content="no-cache"></meta>
<meta http-equiv="content-type" content="text/html; charset=utf-8"></meta>
<link rel="stylesheet" href="../../ct_default.css" type="text/css"></link>
<script type="text/javascript" src="../../jquery-latest.js"></script>
<script type="text/javascript" src="../../jquery.tablesorter.min.js"></script>
</head>
<body>

<br />
<h3>Unexpected I/O</h3>
<pre>

</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:abcast/1 2024-08-16 12:03:35.264 ***</b>
=CRASH REPORT==== 16-Aug-2024::12:03:35.264499 ===
  crasher:
    initial call: gen_server_SUITE:init/1
    pid: &lt;0.244415.0&gt;
    registered_name: my_test_name
    exception exit: {&lt;0.244413.0&gt;,stopped}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2304)
    ancestors: [&lt;0.244413.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: false
    status: running
    heap_size: 1598
    stack_size: 29
    reductions: 3437
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-08-16 12:03:37.365 ***</b>
=ERROR REPORT==== 16-Aug-2024::12:03:37.365429 ===
** Generic server my_test_name terminating 
** Last message in was stop
** When Server state == {formatted,[]}
** Reason for termination ==
** stopped
** Client <0.244439.0> stacktrace
** [{gen_server,mc_recv,5,[{file,"gen_server.erl"},{line,1805}]},
    {gen_server,multi_call,4,[{file,"gen_server.erl"},{line,1766}]},
    {gen_server_SUITE,multicall,1,[{file,"gen_server_SUITE.erl"},{line,1487}]},
    {test_server,ts_tc,3,[{file,"test_server.erl"},{line,1794}]},
    {test_server,run_test_case_eval1,6,[{file,"test_server.erl"},{line,1303}]},
    {test_server,run_test_case_eval,9,[{file,"test_server.erl"},{line,1235}]}]


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-08-16 12:03:37.366 ***</b>
=CRASH REPORT==== 16-Aug-2024::12:03:37.366055 ===
  crasher:
    initial call: gen_server_SUITE:init/1
    pid: <0.244443.0>
    registered_name: my_test_name
    exception exit: stopped
      in function  gen_server:handle_msg/6 (gen_server.erl, line 2254)
    ancestors: [<0.244439.0>]
    message_queue_len: 0
    messages: []
    links: [<0.244439.0>]
    dictionary: []
    trap_exit: false
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 6882
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2024-08-16 12:04:15.902 ***</b>
=ERROR REPORT==== 16-Aug-2024::12:04:15.901907 ===
** Generic server hurra terminating 
** Last message in was {'EXIT',&lt;0.248791.0&gt;,
                               {#Ref&lt;0.4171024481.321388545.224000&gt;,1005738,
                                ok,
                                [{gen_server_SUITE,spec_init}],
                                []}}
** When Server state == {formatted,{}}
** Reason for termination ==
** {#Ref&lt;0.4171024481.321388545.224000&gt;,1005738,ok,
    [{gen_server_SUITE,spec_init}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2024-08-16 12:04:15.902 ***</b>
=CRASH REPORT==== 16-Aug-2024::12:04:15.902281 ===
  crasher:
    initial call: gen_server_SUITE:spec_init_global_default_timeout/2
    pid: &lt;0.248812.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.4171024481.321388545.224000&gt;,1005738,ok,
                     [{gen_server_SUITE,spec_init}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2127)
    ancestors: [&lt;0.248791.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 4756
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init_local_registered_parent/1 2024-08-16 12:04:15.925 ***</b>
=CRASH REPORT==== 16-Aug-2024::12:04:15.925413 ===
  crasher:
    initial call: gen_server_SUITE:spec_init_local/2
    pid: &lt;0.248817.0&gt;
    registered_name: my_server
    exception exit: {&lt;0.248815.0&gt;,stopped}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2304)
    ancestors: [foobar]
    message_queue_len: 0
    messages: []
    links: [&lt;0.248815.0&gt;]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 3482
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during log_mf_h_SUITE:test/1 2024-08-16 12:08:54.042 ***</b>
=CRASH REPORT==== 16-Aug-2024::12:08:54.042656 ===
  crasher:
    initial call: gen_event:init_it/6
    pid: &lt;0.254575.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.4171024481.322961409.65819&gt;,7909,
                     {ok,3},
                     [{log_mf_h_SUITE,test}],
                     []}
      in function  gen_event:terminate_server/4 (gen_event.erl, line 1522)
    ancestors: [&lt;0.254573.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 6772
    stack_size: 29
    reductions: 28457
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-08-16 12:09:03.381 ***</b>
=ERROR REPORT==== 16-Aug-2024::12:09:03.381244 ===
** Generic server <0.255855.0> terminating 
** Last message in was {nodedown,'peer_down_crash-376709-11794@09c889ef6b07'}
** When Server state == {peer_state,
                            #{args => ["-connect_all","false"],
                              name => "peer_down_crash-376709-11794",
                              peer_down => crash},
                            'peer_down_crash-376709-11794@09c889ef6b07',
                            [47,98,117,105,108,100,114,111,111,116,47,111,116,
                             112,47,69,114,108,97,110,103,32,8709,8868,8478,
                             47,101,114,116,115,45,49,53,46,48,46,49,47,98,
                             105,110,47,101,114,108],
                            ["-sname","peer_down_crash-376709-11794",
                             "-connect_all","false","-detached",
                             "-peer_detached","-user","peer","-origin",
                             "g1h3GHRlc3Rfc2VydmVyQDA5Yzg4OWVmNmIwNwAD528AAAAAZr8hlw=="],
                            undefined,undefined,<<>>,
                            {down,
                                {nodedown,
                                    'peer_down_crash-376709-11794@09c889ef6b07'}},
                            {<0.255846.0>,#Ref<0.4171024481.315883524.73957>},
                            0,#{}}
** Reason for termination ==
** {nodedown,'peer_down_crash-376709-11794@09c889ef6b07'}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during dist in peer_SUITE 2024-08-16 12:09:03.382 ***</b>
=CRASH REPORT==== 16-Aug-2024::12:09:03.381871 ===
  crasher:
    initial call: peer:init/1
    pid: <0.255855.0>
    registered_name: []
    exception exit: {nodedown,'peer_down_crash-376709-11794@09c889ef6b07'}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2304)
    ancestors: [<0.255846.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 11207
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-08-16 12:09:04.353 ***</b>
=WARNING REPORT==== 16-Aug-2024::12:09:03.351563 ===
Process <0.91.0> on node 'peer_SUITE-shutdown_stop-547490-11794@09c889ef6b07' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-08-16 12:09:04.367 ***</b>
=WARNING REPORT==== 16-Aug-2024::12:09:03.366500 ===
Process <0.91.0> on node 'peer_SUITE-shutdown_halt-547458-11794@09c889ef6b07' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-08-16 12:09:04.795 ***</b>
=WARNING REPORT==== 16-Aug-2024::12:09:04.795448 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-451043-11794@09c889ef6b07'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-08-16 12:09:04.850 ***</b>
=WARNING REPORT==== 16-Aug-2024::12:09:04.850344 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-547554-11794@09c889ef6b07'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-08-16 12:09:21.271 ***</b>
=WARNING REPORT==== 16-Aug-2024::12:09:20.270366 ===
Process <0.88.0> on node 'peer_SUITE-shutdown_stop-377221-11794@09c889ef6b07' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-08-16 12:09:21.590 ***</b>
=WARNING REPORT==== 16-Aug-2024::12:09:21.590270 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-548386-11794@09c889ef6b07'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-08-16 12:09:21.681 ***</b>
=WARNING REPORT==== 16-Aug-2024::12:09:21.681332 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-548450-11794@09c889ef6b07'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-08-16 12:09:24.376 ***</b>
=ERROR REPORT==== 16-Aug-2024::12:09:24.375955 ===
** Generic server <0.256219.0> terminating 
** Last message in was {'EXIT',<0.256212.0>,
                               {#Ref<0.4171024481.316145666.106295>,1553358,
                                ok,
                                [{peer_SUITE,cntrl_channel_handler_crash}],
                                []}}
** When Server state == {peer_state,#{name =>
                                          "peer_SUITE-cntrl_channel_handler_crash-548802-11794",
                                      connection => standard_io},
                                    'peer_SUITE-cntrl_channel_handler_crash-548802-11794@09c889ef6b07',
                                    [47,98,117,105,108,100,114,111,111,116,47,
                                     111,116,112,47,69,114,108,97,110,103,32,
                                     8709,8868,8478,47,101,114,116,115,45,49,
                                     53,46,48,46,49,47,98,105,110,47,101,114,
                                     108],
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash-548802-11794",
                                     "-user","peer"],
                                    #Port<0.3122>,undefined,<<>>,running,
                                    {<0.256212.0>,
                                     #Ref<0.4171024481.322961409.83237>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.4171024481.316145666.106295>,1553358,ok,
    [{peer_SUITE,cntrl_channel_handler_crash}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2024-08-16 12:09:24.376 ***</b>
=CRASH REPORT==== 16-Aug-2024::12:09:24.376545 ===
  crasher:
    initial call: peer:init/1
    pid: <0.256219.0>
    registered_name: []
    exception exit: {#Ref<0.4171024481.316145666.106295>,1553358,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2127)
    ancestors: [<0.256212.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3122>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 14080
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-08-16 12:09:25.360 ***</b>
=WARNING REPORT==== 16-Aug-2024::12:09:24.358639 ===
Process <0.88.0> on node 'peer_SUITE-shutdown_stop-548898-11794@09c889ef6b07' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-08-16 12:09:25.469 ***</b>
=WARNING REPORT==== 16-Aug-2024::12:09:25.469479 ===
peer:stop() timed out waiting for disconnect from node 'init_debug-451331-11794@09c889ef6b07'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-08-16 12:09:25.775 ***</b>
=WARNING REPORT==== 16-Aug-2024::12:09:25.775333 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-451395-11794@09c889ef6b07'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-08-16 12:09:25.812 ***</b>
=WARNING REPORT==== 16-Aug-2024::12:09:25.812260 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-548834-11794@09c889ef6b07'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-08-16 12:09:26.847 ***</b>
=ERROR REPORT==== 16-Aug-2024::12:09:26.847573 ===
** Generic server <0.256246.0> terminating 
** Last message in was {'EXIT',<0.256190.0>,
                           {#Ref<0.4171024481.315883523.236452>,4028397,ok,
                            [{peer_SUITE,
                                 cntrl_channel_handler_crash_old_release}],
                            []}}
** When Server state == {peer_state,#{args =>
                                          ["-setcookie",
                                           "YZQFRQEGZBPHJBTCYHCR","-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                           "-env","ERL_CRASH_DUMP",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@09c889ef6b07.2024-08-16_11.53.26/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-451363-11794",
                                           "-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@09c889ef6b07.2024-08-16_11.53.26/make_test_dir.stdlib_test.logs/run.2024-08-16_11.54.10/log_private/"],
                                      env => [{"ERL_AFLAGS",false}],
                                      name =>
                                          "peer_SUITE-cntrl_channel_handler_crash_old_release-451363-11794",
                                      connection => standard_io,
                                      exec =>
                                          "/usr/local/lib/erlang-25.3.2.13/bin/erl",
                                      start_cover => false},
                                    'peer_SUITE-cntrl_channel_handler_crash_old_release-451363-11794@09c889ef6b07',
                                    "/usr/local/lib/erlang-25.3.2.13/bin/erl",
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash_old_release-451363-11794",
                                     "-setcookie","YZQFRQEGZBPHJBTCYHCR",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                     "-env","ERL_CRASH_DUMP",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@09c889ef6b07.2024-08-16_11.53.26/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-451363-11794",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@09c889ef6b07.2024-08-16_11.53.26/make_test_dir.stdlib_test.logs/run.2024-08-16_11.54.10/log_private/",
                                     "-user","peer"],
                                    #Port<0.3143>,undefined,<<>>,running,
                                    {<0.256190.0>,
                                     #Ref<0.4171024481.322961409.83751>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.4171024481.315883523.236452>,4028397,ok,
    [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2024-08-16 12:09:26.848 ***</b>
=CRASH REPORT==== 16-Aug-2024::12:09:26.848356 ===
  crasher:
    initial call: peer:init/1
    pid: <0.256246.0>
    registered_name: []
    exception exit: {#Ref<0.4171024481.315883523.236452>,4028397,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2127)
    ancestors: [<0.256190.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3143>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 17731
    stack_size: 29
    reductions: 18965
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-08-16 12:20:08.374 ***</b>
=ERROR REPORT==== 16-Aug-2024::12:20:08.373852 ===
** Generic server <0.302931.0> terminating 
** Last message in was {'EXIT',<0.302929.0>,
                               {#Ref<0.4171024481.359137281.193279>,350853,
                                [{child1,<0.302938.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.302931.0>,supervisor_SUITE},
                            one_for_all,
                            {[child1],
                             #{child1 =>
                                   {child,<0.302937.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576459151],
                            0,never,supervisor_SUITE,
                            {ok,{{one_for_all,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.4171024481.359137281.193279>,350853,
    [{child1,<0.302938.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-08-16 12:20:08.374 ***</b>
=ERROR REPORT==== 16-Aug-2024::12:20:08.373852 ===
** Generic server <0.302932.0> terminating 
** Last message in was {'EXIT',<0.302929.0>,
                               {#Ref<0.4171024481.359137281.193279>,350853,
                                [{child1,<0.302938.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.302932.0>,supervisor_SUITE},
                            rest_for_one,
                            {[child1],
                             #{child1 =>
                                   {child,<0.302938.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576459151],
                            0,never,supervisor_SUITE,
                            {ok,{{rest_for_one,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.4171024481.359137281.193279>,350853,
    [{child1,<0.302938.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-08-16 12:20:08.374 ***</b>
=CRASH REPORT==== 16-Aug-2024::12:20:08.374434 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.302931.0>
    registered_name: []
    exception exit: {#Ref<0.4171024481.359137281.193279>,350853,
                     [{child1,<0.302938.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2127)
    ancestors: [<0.302929.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 11917
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-08-16 12:20:08.375 ***</b>
=CRASH REPORT==== 16-Aug-2024::12:20:08.374705 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.302932.0>
    registered_name: []
    exception exit: {#Ref<0.4171024481.359137281.193279>,350853,
                     [{child1,<0.302938.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2127)
    ancestors: [<0.302929.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 11917
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_bridge_SUITE:simple_global_supervisor/1 2024-08-16 12:20:51.904 ***</b>
=CRASH REPORT==== 16-Aug-2024::12:20:51.903810 ===
  crasher:
    initial call: supervisor_bridge:supervisor_bridge_SUITE/1
    pid: <0.403089.0>
    registered_name: []
    exception exit: killed
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2127)
    ancestors: [bridge9212,<0.403084.0>]
    message_queue_len: 0
    messages: []
    links: [<0.403090.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 4113
  neighbours:
    neighbour:
      pid: <0.403090.0>
      registered_name: work_supervisor_bridge_SUITE_server
      initial_call: {supervisor_bridge_SUITE,internal_loop_init,1}
      current_function: {supervisor_bridge_SUITE,internal_loop,1}
      ancestors: []
      message_queue_len: 0
      links: [<0.403089.0>]
      trap_exit: false
      status: waiting
      heap_size: 233
      stack_size: 3
      reductions: 9
      current_stacktrace: [{supervisor_bridge_SUITE,internal_loop,1,
                      [{file,"supervisor_bridge_SUITE.erl"},{line,152}]}]

</pre></div>
<pre>
</pre>
<center>
<br /><hr /><p>
<a href="../../../all_runs.html">Test run history
</a>  |  <a href="../../../index.html">Top level test index
</a>  |  <a href="../../../suite.log.latest.html">Latest test result</a>
</p>
<div class="copyright">Copyright &copy; 2024 <a href="http://www.erlang.org">Open Telecom Platform</a><br />
Updated: <!--date-->Fri Aug 16 2024 11:54:10<!--/date--><br />
</div>
</center>
</body>
</html>
